README.md

```diff
@@ -8,13 +8,13 @@
 [![Chat on Slack](https://img.shields.io/badge/chat-on%20Slack-brightgreen.svg)](https://join.slack.com/t/rasgousergroup/shared_invite/zt-nytkq6np-ANEJvbUSbT2Gkvc8JICp3g)
 
 # RasgoQL
+RasgoQL is a Python package that enables you to easily query and transform data housed in your Data Warehouse directly from your notebook or IDE of choice.
 
-RasgoQL is a Python package that enables you to easily query and transform tables in your Data Warehouse directly from a notebook. 
+You can quickly create new features, sample data, apply complex aggregates... all without having to write SQL!
 
-You can quickly create new features, sample data, apply complex aggregates... all without having to write SQL! 
-
-Choose from our library of predefined transformations or make your own to streamline the feature engineering process.
+Choose from our library of predefined transformations or make your own to streamline the process of extracting, transforming and loading large amounts of data.
```

Comment:
I think we should stay away from ETL language here. Def sets us up for unrealistic expectations if we brand this as an ETL package